classes/plugins/importexport/PKPImportExportDeployment.inc.php

```diff
@@ -212,9 +212,14 @@ function setFileDBIds($fileDBIds) {
 	 * @param $revisionId integer
 	 * @return integer
 	 */
-	function getFileDBId($fileId, $revisionId) {
-		if (array_key_exists($fileId, $this->_fileDBIds) && array_key_exists($revisionId, $this->_fileDBIds[$fileId])) {
-			return $this->_fileDBIds[$fileId][$revisionId];
+	function getFileDBId($fileId, $revisionId = null) {
+		if ($revisionId) {
+			if (array_key_exists($fileId, $this->_fileDBIds) && array_key_exists($revisionId, $this->_fileDBIds[$fileId])) {
+				return $this->_fileDBIds[$fileId][$revisionId];
+			}
+		} else {
+			// the DB Id is unique for a fileId
+			return current($this->_fileDBIds[$fileId]);
```

Comment:
It looks like "index not found" checking is present when revisions are specified, but not when only the file ID is specified. I'd suggest keeping consistent either way.